custom_dc/faq.md

```diff
@@ -21,9 +21,9 @@ Note that you cannot apply fine-grained access restrictions, such as access to s
 
 ### Will my data or queries end up in base Data Commons? {#data-security}
 
-Your user queries, observations data, or property values are never transferred to base Data Commons. The NL model built from your custom data lives solely in your custom instance. The custom Data Commons Mixer does make API calls to the base Data Commons Mixer (as depicted in [this diagram](/custom_dc/index.html#system-overview)) only in the following instances:
-- At data load time, an API call is made from the custom instance to the base instance to resolve entity names to DCIDs; for example, if your data refers to a particular country name, the custom instance will send an API request to look up its DCID. 
-- At run time, when a user enters an NL query, the custom instance uses its NL model to identify the relevant statistical variables. The custom instance then issues two SQL queries for the statistical variable observations: one to your database and one to the base Data Commons database. These queries only include DCIDs and contain no information about the original query or context of the user request. The data is joined by entity DCIDs.
+Your user queries, observations data, or property values are never transferred to base Data Commons. The NL model built from your custom data lives solely in your custom instance. The custom Data Commons instance does make API calls to the base Data Commons instance (as depicted in [this diagram](/custom_dc/index.html#system-overview)) only in the following instances:
+- At data load time, an API call is made from the custom instance to the base instance to resolve entity names to [DCIDs](/glossary.html#dcid); for example, if your data refers to a particular country name, the custom instance will send an API request to look up its DCID. 
+- At run time, when a user enters an NL query, the custom instance uses its NL model to identify the relevant statistical variables. The custom instance then issues two requests for statistical variable observations: a SQL query to your custom SQL database and an API call to the base Data Commons database. These requests only include DCIDs and contain no information about the original query or context of the user request. The data is joined by entity DCIDs.
 - At run time, when the website frontend renders a data visualization, it will also make the same two SQL queries to get observations data.
```

Comment:
"same two SQL queries" should be changed to something like "same two requests mentioned above"

Reply:
USes its "LOCAL" NL model to identify the relevant ...

At Run time - again,  do we want to be explicit that the data itself is not shared with our frontend visualization work - ie all the rendering happens locally?

Reply:
added "local". 
Re the second comment, I fixed one thing, but I'm not sure what you mean by "shared with frontend visualization work"; I mean I'm guessing there is JS that runs in the browser that a) fetches the data and b) renders the visualizations. What exactly do you want me to say here?